### PR TITLE
last activity filter

### PR DESCRIPTION
**Pull request by msokoloff1**
* Filter exports by all activity related to the label

### CODE REVIEW

File: labelbox/schema/project.py [hunk 1]
```diff
@@ -355,6 +365,7 @@ def _validate_datetime(string_date: str) -> bool:
         """ % (id_param, id_param, filter_param)
 
         start_time = time.time()
+        print(query_str)
```
Comment (vbrodsky) [inline labelbox/schema/project.py:368]: do you need this print?

File: tests/integration/test_export.py [hunk 2]
```diff
@@ -98,3 +99,47 @@ def test_export_filtered_dates(client,
...
+    task = LabelImport.create_from_objects(client, project.uid,
+                                           f'label-import-{uuid.uuid4()}', data)
+    task.wait_until_done()
+
+    regular_export = project.export_labels(download=True)
+    assert len(regular_export) == 1
+
+    filtered_export = project.export_labels(download=True,
+                                            last_activity_start="2020-01-01")
+    assert len(filtered_export) == 1
```
Comment (vbrodsky) [inline tests/integration/test_export.py:134]: but we are not asserting actual data?

Comment (msokoloff1) [reply tests/integration/test_export.py:134]: This is just a test for filtering

**Review by apollonin — APPROVED** [review]
I assume lb-api is ready for it?

**Pull request merged**